### PR TITLE
docs: contained-list

### PR DESCRIPTION
**Pull request by laurenmrice**
**Closes**
[#3049](https://github.com/carbon-design-system/carbon-website/issues/3049)

**Related**
[#10263](https://github.com/carbon-design-system/carbon/discussions/10263)
[#11422](https://github.com/carbon-design-system/carbon/issues/11442)
[#11969](https://github.com/carbon-design-system/carbon/pull/11969)

- Adds new usage and style documentation for contained list

----

**New**
- New page for Contained list component 
- Usage tab
- Style tab
- Code tab (placeholder)
- Accessibility tab (placeholder)
- Images

### CODE REVIEW

File: src/pages/components/contained-list/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+- A list title is a descriptive name for the group of list items.
+- List titles should be short and clear, ranging from one to three words to
+  describe the list item group.
+- Use sentence-case capitalization.
+
+#### List item
+
+— List item text can vary but the structure of the content should be similar per
+row in the same contained list.
+
```
Comment (aagonzales) [inline src/pages/components/contained-list/usage.mdx:145]: There's a line gap here.
image

**Review by aagonzales — COMMENTED** [review]
Looking good! Most of my comments are just suggestions. 

I'm wondering since these lists are so use case dependent if we should add in more of the mini-component examples to help explain where to use them. _A cherry on top but I don't think its necessary to merge._ 

image

File: src/pages/components/contained-list/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+![Contained list anatomy](images/contained-list-usage-1.png)
+
+</Column>
+</Row>
+
+1. **List header:** Area to group the succeeding list items.
+2. **List title:** Descriptive name for the group of list items.
+3. **List item:** The content with in a list item row.
+4. **Interactive element:** An element that you can interact with or treat as an
```
[the remark this answers is not in the document]
Comment (laurenmrice) [reply src/pages/components/contained-list/usage.mdx:78]: Optional!

File: src/pages/components/contained-list/style.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+| ---------------- | ------------------ | -------------- | --------------------- |
+| Title: on page   | 14 / 0.875         | SemiBold / 600 | `$heading-compact-01` |
+| Title: disclosed | 12 / 0.75          | Regular / 400  | `$label-01`           |
+| Item             | 14 / 0.875         | Regular / 400  | `$body-01`            |
+
+## Structure
+
+| Element           | Property                    | px / rem | Spacing token |
+| ----------------- | --------------------------- | -------- | ------------- |
+| Header: on page   | height                      | 32/2     | `$spacing-07` |
```
Comment (aagonzales) [inline src/pages/components/contained-list/style.mdx:89]: You need spaces between all the numbers
```suggestion
| Header: on page   | height                      | 32 / 2     | `$spacing-07` |
```

**Comment by laurenmrice** [comment]
@aagonzales Ready for review again! The image on the usage tab for [Alignment](https://carbondesignsystem-git-fork-laurenm-dc957e-carbon-design-system.vercel.app/components/contained-list/usage#alignment) is not showing up correctly on the preview, but it is correct in the file I changed.

**Review by aagonzales — APPROVED** [review]
Looks golden, ship it!

**Review by tay1orjones — APPROVED** [review]
Awesome! 🎉

**Pull request merged**